src/widgets/widgets.ts

```diff
@@ -53,6 +53,7 @@ interface DataType {
 export type IMultiSelectOptionValue = {
   type: 'multi-select';
   defaultValue: string[];
+  info?: boolean;
```

Comment:
Since this is a common attribute over all option types, can you extract it to it's own type and derive it?